server/player/form/submit.go

```diff
@@ -30,6 +30,12 @@ type MenuSubmittable interface {
 // buttons on a Modal form will not have images.
 type ModalSubmittable MenuSubmittable
 
+// Closer represents a form which has special logic when being closed by a Submitter. The form will have its Close method called, when it is closed.
```

Comment:
Would remove the `The form...is closed' sentence as the Close method documents that.